docs/runbooks/HCOGoldenImageWithNoSupportedArchitecture.md

```diff
@@ -0,0 +1,207 @@
+# HCOGoldenImageWithNoSupportedArchitecture
+
+## Meaning
+
+When running on a heterogeneous cluster, a cluster with nodes of different
+architectures, the DataImportCronTemplates (DICTs; also known as golden
+images), in the hyperconverged cluster operator (HCO) should be annotated with
+the `ssp.kubevirt.io/dict.architectures` annotation, where the value is the
+list of the architectures supported by the image, that is defined in each DICT.
+
+For pre-defined DICTs, this annotation is already set, but for custom DICTs
+(user defined DICTs), this annotation must be set by the user in the
+HyperConverged custom resource (CR).
+
+For each DICT, if the annotation does not include any architecture that is
+supported by the cluster (which mean, there is no node in the cluster that with
+any of the architectures listed in the DICT annotation), Then HCO will trigger
```

Comment:
> which mean, there is no node in the cluster that with
any of the architectures listed in the DICT annotation

which mean, there is no node in the cluster with
 the architectures listed in the DICT annotation